src/linenoise/linenoise.c

```diff
@@ -946,9 +971,13 @@ static int linenoiseRaw(char *buf, size_t buflen, const char *prompt) {
         }
     } else {
         /* Interactive editing. */
+#ifndef _WIN32
```

Comment:
Does the address_parser client still work on Windows with all of these conditionals? I didn't write linenoise, but it seems like enableRawMode/disableRawMode are not optional.

Reply:
I've confirmed that the Windows build works just like the *nix build. Unfortunately, I don't have valgrind on MSys2.

Reply:
I've reduced the amount of #ifdef'ery and amended the commit.